apple/Handlers/RNPanHandler.m

```diff
@@ -123,6 +123,7 @@ - (void)interactionsMoved:(NSSet *)touches withEvent:(UIEvent *)event
 
   if (self.state == UIGestureRecognizerStatePossible && [self shouldFailUnderCustomCriteria]) {
     self.state = UIGestureRecognizerStateFailed;
+    [self triggerAction];
```

Comment:
I assume that it should be here, but to clarify - should it?

Reply:
Yeah, it's loosely related to this - the fail event here wouldn't be sent until `reset` gets called by the OS, which happens after the finger is lifted from the screen - the gesture could fail, but the information about that would be delayed.